### PR TITLE
May 10

### PR DESCRIPTION
**Pull request by HarshitaAgarwal09**

### CODE REVIEW

**Comment by HarshitaAgarwal09** [comment]
May 10 
Problem : Find the Town Judge
Language: Java

File: Valid Perfect Square.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        
+        while(left<=right)
+        {   long long mid = left + (right-left)/2;
+            if(mid*mid < num) left = mid+1;
+            else if(mid*mid > num) right = mid-1;
+            else return true;
+        }
+        return false;
+    }
+};
```
Comment (manthanchauhan) [inline Valid Perfect Square.cpp:16]: @HarshitaAgarwal09 a cpp code for the same problem already exists..

Comment (HarshitaAgarwal09) [reply Valid Perfect Square.cpp:16]: Sorry, didnt see that.

Comment (HarshitaAgarwal09) [reply Valid Perfect Square.cpp:16]: No, it doesnt exists. Only python code exists

Comment (manthanchauhan) [reply Valid Perfect Square.cpp:16]: sorry, my Bad.

**Pull request merged**